about.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: docs
+title: About
+headline: All About Pico
+description: What *is* a "stupidly simple, blazing fast, flat file CMS" anyway?
+toc:
+  a-flat-file-cms: A Flat File CMS
+  web-pages-without-html:
+    _title: Web Pages Without HTML
+    markdown-makes-formatting-easy: Markdown Makes Formatting Easy
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    theming-with-twig-templates: Theming With Twig Templates
+    plugins-extend-picos-php: Plugins Extend Pico's PHP
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+galleries:
+  standalone:
+    style: magnify
+    images:
+      -
+        heading: Pico's Default Theme
+        description: It's a bit bare... but that's intentional!
+        thumbnail: /style/images/docs/about/thumbnails/default_theme.png
+        fullsize: /style/images/docs/about/fullsize/default_theme.png
+        styles: "float: right; margin-left: 2em;"
+  workflow:
+    headline: Pico's Workflow
+    description: Creating content with Pico couldn't be simpler!
+    style: carousel-box
+    images:
+      -
+        heading: A Pico Installation
+        description: Here's what Pico looks like installed on your web server.
+        thumbnail: /style/images/docs/about/thumbnails/pico_folder.png
+        fullsize: /style/images/docs/about/fullsize/pico_folder.png
+      -
+        heading: Your Content Folder
+        description: An example of a content folder in Pico.  This one is from a writing website.
+        thumbnail: /style/images/docs/about/thumbnails/content_folder.png
+        fullsize: /style/images/docs/about/fullsize/content_folder.png
+      -
+        heading: Pico's Configuration File
+        description: Configuring Pico is as simple as uncommenting a few lines in the included template.
+        thumbnail: /style/images/docs/about/thumbnails/config.png
+        fullsize: /style/images/docs/about/fullsize/config.png
+      -
+        heading: Editing a Page
+        description: Creating content couldn't be easier using Markdown.  Here's an example of editing a page.
+        thumbnail: /style/images/docs/about/thumbnails/editing_markdown.png
+        fullsize: /style/images/docs/about/fullsize/editing_markdown.png
+  themes:
+    headline: Community Themes
+    description: Ready-to-go, community developed themes to customize your Pico website.
+    style: carousel-box
+    images:
+      -
+        heading: NotePaper Theme
+        description: "[NotePaper](http://development.sjmcdougall.com/pico-themes/NotePaper/) - A highly customizable theme, designed exclusively for Pico."
+        thumbnail: /style/images/docs/about/thumbnails/notepaper_theme.jpg
+        fullsize: /style/images/docs/about/fullsize//notepaper_theme.jpg
+      -
+        heading: Simple Sidebar Theme
```

Comment:
Instead of this theme (we already have a sidebar theme with SimpleTwo in the gallery), a screenshot of one of those "card" themes (e.g. Identity) would be great. Also an additional screenshot of a "blog" theme (e.g. "Clean Blog") might be a good idea.